Lib/fontmake/font_project.py

```diff
@@ -306,25 +349,35 @@ def save_otfs(
             finder = partial(_varLib_finder, directory=interpolate_layout_dir,
                              ext=ext)
 
+        compiler_options = dict(
+            useProductionNames=use_production_names,
+            reverseDirection=reverse_direction,
+            cubicConversionError=conversion_error,
+            inplace=True,  # avoid extra copy
+        )
+        if use_afdko:
+            compiler_options["featureCompilerClass"] = FDKFeatureCompiler
+
+        if interpolatable:
+            if not ttf:
+                raise NotImplementedError(
+                    "interpolatable CFF not supported yet")
+
+            logger.info('Building interpolation-compatible TTFs')
+
+            fonts = compileInterpolatableTTFs(ufos, **compiler_options)
+        else:
+            fonts = self._iter_compile(
+                ufos,
+                ttf,
+                removeOverlaps=remove_overlaps,
+                optimizeCFF=subroutinize,
+                roundTolerance=cff_round_tolerance,
+                **compiler_options)
+
         do_autohint = ttf and autohint is not None
-        for ufo in ufos:
-            name = self._font_name(ufo)
-            logger.info('Saving %s for %s' % (ext.upper(), name))
-
-            if use_production_names is None:
-                use_production_names = not ufo.lib.get(
-                    GLYPHS_PREFIX + "Don't use Production Names")
```

Comment:
This Glyphs.app-specific UFO lib key will also be supported in the upcoming ufo2ft
https://github.com/googlei18n/ufo2ft/commit/2ca438c0e7779b092f44f204d7bda666bb9af68b